templates/cluster-template-powervs.yaml

```diff
@@ -103,10 +103,10 @@ spec:
               env:
               - name: vip_arp
                 value: "true"
-              - name: vip_interface
-                value: "env2"
               - name: port
                 value: "6443"
+              - name: vip_interface
+                value: env2
```

Comment:
env2 -> "env2" just a observation, if both works fine then we are good.

Reply:
yes, its not really required.